File: exercise-007/exercises/java/Circle.java
package exercises.java;

public class Circle {

    private double radius;
    private String color;


    public Circle() {

    }

    public Circle(double radius) {
        this.radius = radius;
    }


    public double getRadius() {
        return radius;
    }

    public void setRadius(double radius) {
        this.radius = radius;
    }

    public String getColor() {
        return color;
    }

    public void setColor(String color) {
        this.color = color;
    }


    public double getArea() {
        return Math.pow(radius, 2) * Math.PI;
    }
}

File: exercise-007/Exercise7/CSharp/Circle.cs
using System;

namespace Exercise7.CSharp
{


    class Circle
    {

        private double radius;
        private string color;

        public Circle()
        {

        }

        public Circle(double radius)
        {
            this.radius = radius;
        }


        public void setColor(string color)
        {
            this.color = color;
        }

        public void setRadius(double radius)
        {
            this.radius = radius;
        }

        public string getColor()
        {
            return this.color;
        }

        public double getRadius()
        {
            return this.radius;
        }

        public double getArea()
        {
            return Math.Pow(this.radius, 2) * Math.PI;
        }
    }
}